app/code/Magento/Eav/Test/Unit/Model/Api/SearchCriteria/CollectionProcessor/FilterProcessorTest.php

```diff
@@ -5,14 +5,18 @@
  */
 namespace Magento\Eav\Test\Unit\Model\Api\SearchCriteria\CollectionProcessor;
 
+use InvalidArgumentException;
 use Magento\Eav\Model\Api\SearchCriteria\CollectionProcessor\FilterProcessor;
 use Magento\Framework\Api\Filter;
 use Magento\Framework\Api\Search\FilterGroup;
 use Magento\Framework\Api\SearchCriteria\CollectionProcessor\FilterProcessor\CustomFilterInterface;
 use Magento\Framework\Api\SearchCriteriaInterface;
 use Magento\Framework\Data\Collection\AbstractDb;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use stdClass;
 
-class FilterProcessorTest extends \PHPUnit\Framework\TestCase
+class FilterProcessorTest extends TestCase
 {
     /**
      * Return model
```

Comment:
Please move the private method `getModel` to the bottom of the class. The private and protected methods should not be followed by public ones.